q2_emperor/_plot.py

```diff
@@ -59,7 +59,15 @@ def generic_plot(output_dir: str, master: skbio.OrdinationResults,
 
 def plot(output_dir: str, pcoa: skbio.OrdinationResults,
          metadata: qiime2.Metadata, custom_axes: str = None,
-         ignore_missing_samples: bool = False) -> None:
+         ignore_missing_samples: bool = False,
+         ignore_pcoa_features: bool = True) -> None:
```

Comment:
Should the default be `False`? I think it would be more polite to users to explode first, rather than removing data.

Reply:
Yes, great suggestion!